backend/test/acceptance/__snapshots__/config.spec.js.snap

```diff
@@ -1,5 +1,30 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`config should reject unauthenticated requests 1`] = `Array []`;
+
+exports[`config should reject unauthenticated requests 2`] = `
+Object {
+  "code": 401,
+  "details": Object {
+    "name": "UnauthorizedError",
+    "stack": "UnauthorizedError: No authorization token was found
```

Comment:
A stack trace in a snap is not a good idea

Reply:
✅